ramls/time-period.json

```diff
@@ -18,11 +18,25 @@
         "Months"
       ],
       "default" : "Days"
+    },
+    "closeLibraryDateManagement": {
+      "type": "string",
+      "description": "enum for closeLibraryDateManagement",
+      "enum":[
+        "Keep_the_current_due_date",
+        "Move_to_the_end_of_the_previous_open_day",
+        "Move_to_the_end_of_the_next_open_day",
+        "Keep_the_current_due_date_time",
+        "Move_to_end_of_current_service_point_hours",
+        "Move_to_beginning_of_next_open_service_point_hours"
+      ],
+      "default" : "Keep_the_current_due_date"
     }
   },
   "required": [
     "duration",
-    "intervalId"
+    "intervalId",
+    "closeLibraryDateManagement"
```

Comment:
A new required property is a compatibility breaking change. 

That should be represented by a change to the major version of the interface(s) affected by the change (in both the module descriptor and the RAML documentation).

And a major version change in the module implementation version (in the pom.xml)